default.nix

```diff
@@ -354,18 +360,17 @@ rec {
   commonQemuOptions = ''
     -only-migratable \
     -nographic -no-reboot \
-    -cpu IvyBridge \
-    -enable-kvm \
-    -net none -m "$mem" \
+    -cpu host -enable-kvm \
```

Comment:
`-cpu host` is a portability hazard. If you build the suspension on a different CPU than you intend to restore it on (quite possible, with Nix deployments from your dev setup to a server), it will error. The `IvyBridge` value is of course irrelevant, that's just the server I had available.

Reply:
Hmm, I tried migrating from an AMD Ryzen 5 4650U to an Intel Core i5-2520M, it seems like the only things that work are to disable KVM or to use `-cpu qemu64`. The latter seems reasonable